lib/generators/shopify_app/install/templates/shopify_session_repository.rb

```diff
@@ -1,23 +0,0 @@
-# You should replace InMemorySessionStore with what you will be using
```

Comment:
Its fairly well documented in the Readme itself. I'll read through this and make sure its all captured